docs/faqs.md

```diff
@@ -463,3 +465,11 @@ To have more context about it, please, visit issues #11684 and #11022
 In summary, we do not recommend `full_package_mode` or any other custom package id mode for requirements on CCI, it will break other PRs soon or later.
 Instead, prefer using `shared=True` by default, when needed.
 Also, when having a similar situation, do not hesitate in opening an issue explaining your case, and ask for support from the community.
+
+## Do ConanCenter recipes support editable mode?
+
+No, as they download source code the expectations is that it should not require development - which is the purpose of editable mode. 
```

Comment:
I don't understand that statement. how exactly `download source code` implies `should not require development`? don't you use source code for development?
if they downloaded binaries, that statement could made sense, but otherwise...